docs/modules/ROOT/nav.adoc

```diff
@@ -318,10 +319,7 @@
 ***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates/ssl-options.adoc[ssl-options]
 ***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/SSL Certificates/ssl-certificates.adoc[ssl-certificates]
 **** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home[Phone Home]
-***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/phonehome-overview.adoc[phonehome-overview]
-***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/phone-home-information.adoc[phone-home-information]
-***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/phone-home-asadmin.adoc[phone-home-asadmin]
-***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/disabling-phone-home.adoc[disabling-phone-home]
+***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/Phone Home/PhoneHome.adoc[PhoneHome]
```

Comment:
It's Phone Home (with a space) and not PhoneHome, And it doesn't have to be in a separate directory Phone Home. So, in the end, it should be (and the file moved to the Configuration Options directory):
```suggestion
***** xref:Technical Documentation/Payara Server Documentation/Server Configuration And Management/Configuration Options/PhoneHome.adoc[Phone Home]
```